gpflow/experimental/check_shapes/__init__.py

```diff
@@ -55,6 +55,9 @@ def linear_model(
     * Use ``.<name>`` to refer to attributes of the object.
     * Use ``[<index>]`` to refer to elements of a sequence. This is particularly useful if your
       function returns a tuple of values.
+    * Use ``[all]`` to select all elements in a collection.
+    * Use ``.keys()`` to select all keys in a mapping.
+    * Use ``.values()`` to select all values in a mapping.
```

Comment:
I'm not sure how happy I am about the syntax here. Any better ideas?

Reply:
I haven't used `check_shapes` yet, so it is hard to say what is best in that case. I will leave it to you to decide.